x-pack/plugins/cloud_security_posture/public/pages/configurations/configurations.test.tsx

```diff
@@ -61,7 +61,7 @@ const renderFindingsPage = () => {
         licensing: licensingMock.createStart(),
       }}
     >
-      <Findings />
+      <Configurations />
```

Comment:
What do you think about `ConfigurationsFindings` and `VulnerabiltiesFindings` for their index files? its a bit longer but i think it conveys that those are findings pages better. just reading `configurations` makes me expect a page with switches and toggles to change my settings 😅